lib/model/codeceptjs-factory.js

```diff
@@ -67,29 +66,14 @@ module.exports = new class CodeceptjsFactory {
     });
   }
 
-  createRunner() {
-    config.load(path.join(TestProject, 'codecept.conf.js'))
-    const cfg = config.get();
-
-    // create runner
-    let codecept = new Codecept(config, opts);
-
-    // initialize codeceptjs in current dir
-    codecept.initGlobals(TestProject);
-
-    // create helpers, support files, mocha
-    Container.create(config, opts);
-
-    // initialize listeners
-    codecept.runHooks();
-
-    // run bootstrap function from config
-    codecept.runBootstrap();
-
-    // load tests
-    codecept.loadTests(cfg.tests);
-
-    // run tests
-    codecept.run();
+  runWebHooks() {
+    disableTestRetries();
   }
+
 };
+
+function disableTestRetries() {
+  event.dispatcher.on(event.test.before, (test) => {
+    test.retries(0);
```

Comment:
Can I call test.retries(0) in the helper? Will that work? Because if it does I would rather do it there, since it should contain all the special logic for running in web ui.

Reply:
Good idea. Probably, yes, this would be possible